x/ibc/02-client/keeper/client.go

```diff
@@ -50,15 +52,27 @@ func (k Keeper) CreateClient(
 }
 
 // UpdateClient updates the consensus state and the state root from a provided header
-func (k Keeper) UpdateClient(ctx sdk.Context, clientID string, header exported.Header) error {
+func (k Keeper) UpdateClient(
```

Comment:
@cwgoes we need to update the spec

Reply:
Beyond this PR - https://github.com/cosmos/ics/pull/367?